snap/snapcraft.yaml

```diff
@@ -63,6 +63,13 @@ description: |
 
   The DCGM snap does not currently include the performance testing tool `dcgmproftester` in order to limit the size of the snap.
 
+  **Cryptography**
+  ---
+
+  The snap installs the CUDA keyring, verifying its integrity using `curl` and SHA256 checksums.
```

Comment:
careful that this makes it sound like the snap is going to deploy the keyring and the new source on the host system